docs/user/installation.rst

```diff
@@ -168,40 +168,40 @@ Navigate to your preferred installation directory, i. e. use ``mkdir ~/intelmq &
 You need to prepare some volumes & configs. Edit the left-side after -v, to change paths.
```

Comment:
Please mention that the `example_config` folder the docker instructions below refer to can be found in the `intelmq-docker` repository. Or adjust the documentation to the paragraph about docker-compose where the user is instructed to clone the repository in the first step, to get all the needed files.

Reply:
How was this resolved?

Reply:
I still don't see how this was resolved. Am I missing something?

Reply:
You mention the usage without docker-compose but the ``intelmq-docker`` repository is built for docker-compose usage. I will add that you need to clone the repository too for building & testing, but for installation its not required as you download all images directly from dockerhub. It's the not recommended way.

Reply:
But in your `code-block` you give the example with `-v ~/intelmq/example_config/intelmq/etc/:/opt/intelmq/etc/ ` for **running** the docker image. The command **fails** without the `example_folder`- so either you explain to the user how to create the `example_folder` and all the needed files for **running** the docker image successfully *or* you mention that the repository needs to be cloned.

Reply:
Easier solution is to clone the repository as you can switch to `docker-compose` instead if you want. Added the clone command now to documentation for non `docker-compose` usage too.